src/Actions/Login.php

```diff
@@ -15,6 +15,7 @@ class Login
 	public static function init( $form_id ) {
 		if ( Utils::is_active() && ! empty( $form_id ) ) {
 			add_action( 'gform_validation_' . $form_id, [ __CLASS__, 'validation' ] );
+			add_filter( 'gform_confirmation_' . $form_id, [ __CLASS__, 'confirmation' ], 10, 1 );
```

Comment:
These are the default params `10, 1`  So there is no need to put them at the end, Same as the above.